rsts/community/troubleshoot.rst

```diff
@@ -112,7 +112,11 @@ Troubles With Inconsistent Names for Pods and Downstream Resources
 - Flyte uses the format ``executionid-node-id-attempt`` from the node to assign a name to a Kubernetes pod or downstream resource.
 - But if this is an invalid name for a Kubernetes pod, Flyte assigns a valid name of random characters instead.
 
+Troubles with handling large responses in ``FlyteRemote.sync`` 
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
+- ``Received message larger than max (xxx vs. 4194304)`` usually crops up when the message size is too large.
+- To fix this, edit the flyte-admin-base-config config map to increase ``maxMessageSizeBytes`` value.
```

Comment:
Can you also add the kubectl command mentioned in the slack conversation?

Reply:
OK, I'll do that, but I'll also add the `^^^` to fix the error, I'm sorry I didn't notice that before

Reply:
There's a command mentioned in the conversation I included in the PR description, right? Can you include it?

Reply:
You can say *** is how you can edit the flyte-admin-base-config config map.

Reply:
Yup I include it